### ISSUE TITLE
Maybe we can get old policy's logits through sampling process

### ISSUE THREAD

**Issue by zhangyanming-cs** [Gen-Verse/dLLM-RL#17]
https://github.com/Gen-Verse/dLLM-RL/blob/7e4f5f1afefc0faf56197980840d95156344b58a/train/rl_trado.py#L630

Hi there. I noticed that you use this function to compute the old policy's logits. But I think maybe we can get the logits directly through sampling process, given that the model's weights don't change between sampling and the beginning of training.
